index.html

```diff
@@ -24,125 +41,629 @@
+          />
+          <path
+            class="logomark"
+            d="M143.1 28.7c-3.8-1.7-14.6-6.6-19.6-4.9-3.4 1.1-7.3 9.4-9.9 11.8-.3.2-.6.5-1 .8-7.1-13.4-20.8-21-29.9-18.5-6.2 1.7-11.9 8.9-14.5 18.1-4.5-2.7-6.8-7.2-10.8-11.3-3.1-3.2-7.3-1.9-10.9-.2-7.2 3.3-16.6 8.2-16.3 15.9 4.2-3 8.3-2.1 9.6 3.4.3 1 2.6-3.6 5.4-4.6 3.2-1.2 6.7 1.7 7.4 4.9.2.6 2.9-3.5 5.7-2.1 2 .9 3.2 5.1 3.9 4.7 2-1.3 3.4-2.8 4.8-4-.5 4.7-.2 9.7 1.2 14.7.1.3.2.6.2.8 1-8.9 5.9-16.2 11.4-17.7 7.7-2.2 20.1 5.8 23.8 18.9 2.7 9.7.8 19.3-4.2 24.6.6-.1 1.2-.2 1.8-.4 13.1-3.7 19.6-21.4 14.5-39.5-.1-.4-.2-.7-.3-1 3.2 1.7 4.7 4.5 4.8 4.1 1.2-5.6 7.9-5.7 12.3-1.7.4.4.6-4.3 2.3-4.9 2.7-1 5.7.5 8.5 2.2.5.3.4-2.2.3-2.5-.2-1.2-.1-2.5 1.3-2.9 2.1-.6 4.5.5 6.4.9-.5-4.5-4.6-8-8.2-9.6z"
+          />
+        </svg>
+      </a>
+      <nav>
+        <ul data-navlist>
           <li>
-            <button popovertarget="popover11-1" popovertargetaction="show">
-              Items
-            </button>
+            <a
+              data-nav
+              href="https://html.spec.whatwg.org/multipage/popover.html#dom-popover"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Spec</a
+            >
+          </li>
+          <li>
+            <a
+              data-nav
+              href="https://developer.mozilla.org/en-US/docs/Web/API/Popover_API"
+              target="_blank"
+              rel="noopener noreferrer"
+              >API</a
+            >
+          </li>
+          <li>
+            <a
+              data-nav
+              href="https://github.com/oddbird/popover-polyfill"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Source</a
+            >
           </li>
         </ul>
-        <div id="popover11-1" popover="auto" style="left: 200px">
-          <ul>
-            <li><button>Nested</button></li>
-            <li><button>Menu</button></li>
-            <li><button>Items</button></li>
+      </nav>
+    </header>
+    <main>
+      <header class="main-header">
+        <h1 id="title">
```

Comment:
Do we want to enforce some padding here? At certain screen widths (e.g. ~970px) this gets slightly cut off.

![Screenshot 2024-04-24 at 2 43 17 PM](https://github.com/oddbird/popover-polyfill/assets/552316/5b84f977-4158-4ee1-a73f-7b14be6e0579)

Reply:
@jgerigmeyer I adjusted the layout breakpoint and padding so this is no longer overflowing. https://github.com/oddbird/popover-polyfill/pull/187/commits/27c0dce712b7d58125a42a4b735369003cca4984